### ISSUE TITLE
USWDS - Footer: Add autocomplete attribute to email input

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#6001]
## Summary
The email input in the big footer variant needs the [autocomplete="email"](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) attribute in order to meet [WCAG criteria 1.3.5](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html). 

This should be fixed quickly because the footer checklist item is currently marked as "passed with exceptions". We should update that checklist item to "pass" when this issue is complete.
